apache-rat-core/src/main/java/org/apache/rat/Report.java

```diff
@@ -160,7 +160,7 @@ public String desc() {
      * Name of File to exclude from report consideration.
      */
     static final Option EXCLUDE_CLI = Option.builder("e").longOpt("exclude").hasArgs().argName("Expression")
-            .desc("Excludes files matching wildcard <expression>. May be followed by multiple arguments. "
+            .desc("Excludes files matching wildcard <Expression>. May be followed by multiple arguments. "
```

Comment:
Pls update the site template to keep its text in sync with the stuff in the code. Should we add a javadoc in this class on where to keep it in sync with in case of changes?